### PR TITLE
#163921784 Admin user should be able to Create a political party. 

### PR DESCRIPTION
**Pull request by Nduhiu17**
### What does this PR do?
Have the end point to create a political party working

### Description of Task to be completed?
Admin user should create a political party

### How should this be manually tested?
* Clone the repo by git clone by : git clone https://github.com/Nduhiu17/politico-server.git
* Navigate into the project by: cd political-server
* Install a virtual environment by: python3.6 -m venv virtual
* Install dependancies by: pip install -r requirements.txt
* Set server's secret key by:
  export SECRET_KEY='put-your-secret-key-here'
* Export your database url by:
  export DATABASE_URL="postgres://yourusername:yourpassword@localhost:5432/your-db-name"
* Run the server by: python manage.py server
* Hit the following endpoint with postman: [http://127.0.0.1:5000/api/v2/parties['POST']]

### What are the relevant pivotal tracker stories?
[163921784](https://www.pivotaltracker.com/story/show/163921784)

### Screenshot
When admin has logged in
screenshot from 2019-02-19 01-50-50

When non admin has logged in
screenshot from 2019-02-18 22-16-56

### CODE REVIEW

**Comment by petermwash** [comment]
Hey, on your screenshot, I'm curious to know why is the `date created` value shown as `slogan`?

Also under `the relevant pivotal tracker stories`, I would suggest that you don't input the whole URL rather you should use a markdown language to embed the url on the story id. The markdown looks like this `[Story-id-here](Story-link-goes-here)` 
e.g `[163921784](https://www.pivotaltracker.com/story/show/163921784)`

File: app/V2/auth/models.py [hunk 1]
```diff
@@ -76,4 +80,17 @@ def verify_hashed_password(password, hashed_password):
         """Method to verify password with the hashed password"""
         return pbkdf2_sha256.verify(password, hashed_password)
 
+    @classmethod
+    def find_by_nationaid(cls, id):
+        """This method gets a user using email"""
+        try:
+            cursor.execute("select * from users where nationalid = %s", (id,))
+            user = cursor.fetchone()
```
Comment (petermwash) [inline app/V2/auth/models.py:88]: Here the docstrings says that the `method gets a user using email` yet the implementation gets a user using national_id. Kindly correct that.

**Review by petermwash — CHANGES_REQUESTED** [review]

File: app/V2/auth/models.py (hunk 1 above)
Comment (Nduhiu17) [reply app/V2/auth/models.py:88]: Noted.I will correct this

**Comment by Nduhiu17** [comment]
> Hey, on your screenshot, I'm curious to know why is the `date created` value shown as `slogan`?
> 
> Also under `the relevant pivotal tracker stories`, I would suggest that you don't input the whole URL rather you should use a markdown language to embed the url on the story id. The markdown looks like this `[Story-id-here](Story-link-goes-here)`
> e.g `[163921784](https://www.pivotaltracker.com/story/show/163921784)`

Thanks.I will implement these reviews

File: app/V2/auth/models.py (hunk 1 above)
Comment (Nduhiu17) [reply app/V2/auth/models.py:88]: Noted.I will correct this.

**Review by petermwash — APPROVED** [review]
LGTM.

**Pull request merged**